src/04-modality-specific-files/10-microscopy.md

```diff
@@ -0,0 +1,430 @@
+```
+Microscopy data MUST be stored in the `microscopy` directory.
+
+### File formats
+The Microscopy community uses a variety of formats for storing raw data, and there is no single
+standard that all researchers agree on. However, a standardized file structure has been developed
+by the [Open Microscopy Environment](https://www.openmicroscopy.org/) for whole-slide imaging with
+the [OME-TIFF file specifications](https://docs.openmicroscopy.org/ome-model/6.1.2/ome-tiff/file-structure.html).
+The OME-TIFF file allows for multi-page TIFF files to store multiple image planes and supports
+multi-resolution pyramidal tiled images. An OME-XML data block is also embedded inside the
+file’s header.
+
+The BIDS standard accepts microscopy data in different file formats to accommodate datasets
+stored in 2D image formats and whole-slide imaging formats, to accommodate lossless and lossy
+compression, and to avoid unnecessary conversions of the original data from a non-tiled to a
+tiled format, or vice-versa.
+
+For BIDS, Microscopy raw data MUST be stored in one of the following formats:
+
+-   [Portable Network Graphics](http://www.libpng.org/pub/png/) (`.png`)
+
+-   [Tag Image File Format](https://www.adobe.io/open/standards/TIFF.html) (`.tif`)
+
+-   [OME-TIFF](https://docs.openmicroscopy.org/ome-model/6.1.2/ome-tiff/specification.html#)
+    (`.ome.tif` for standard TIFF files or `.ome.btf` for
+    [BigTIFF](https://www.awaresystems.be/imaging/tiff/bigtiff.html) files)
+
+If different from PNG, TIFF or OME-TIFF, the original unprocessed data in the native format MAY be
+stored in the [`/sourcedata` directory](../02-common-principles.md#source-vs-raw-vs-derived-data).
+
+Future versions may extend this list of supported file formats, for example with the
+Next-Generation File Formats currently developed by OME ([OME-NGFF](https://ngff.openmicroscopy.org/latest/))
+as a successor to OME-TIFF for better remote sharing of large datasets.
+
+### Modality suffixes
+Microscopy data currently support the following imaging modalities:
+<!---
+The following table will be generated automatically with macros after community review.
+A "manual" table is provided to facilitate the review process.
+-->
+| **Name**                                      | `suffix` |
+| --------------------------------------------- | -------- |
+| Transmission electron microscopy              | TEM      |
+| Scanning electron microscopy                  | SEM      |
+| Micro-CT                                      | uCT      |
+| Bright-field microscopy                       | BF       |
+| Dark-field microscopy                         | DF       |
+| Phase-contrast microscopy                     | PC       |
+| Differential interference contrast microscopy | DIC      |
+| Fluorescence microscopy                       | FLUO     |
+| Confocal microscopy                           | CONF     |
+| Polarized-light microscopy                    | PLI      |
```

Comment:
It was brought to our attention that for PLI, you get images from different contrasts from the microscope (i.e. orientation, transmittance image, retardance image and inclination image) for the same acquisition.

I think this case is already covered in our BEP with the `acq-<label>` but I wanted to share my thoughts so everyone can chip in.

The `acq-<label>`, which is described [here](https://bids-specification.readthedocs.io/en/stable/99-appendices/09-entities.html#acq) in BIDS as “a custom label the user MAY use to distinguish a different set of parameters used for acquiring the same modality” with the example “In case different sequences are used to record the same modality (for example, RARE and FLASH for T1w) this field can also be used to make that distinction".

It appears to me that the example matches the PLI case, and that the different PLI acquisitions could be named for example:
`sub-01_sample-01_acq-orientation_PLI.ome.tif`
`sub-01_sample-01_acq-transmittance_PLI.ome.tif`
`sub-01_sample-01_acq-retardance_PLI.ome.tif`
`sub-01_sample-01_acq-inclination_PLI.ome.tif`

Reply:
Hi all. Many thanks for all your efforts on this initiative to date! 

My one query with the PLI solution is that the different PLI maps (retardance, transmittance, orientation, inclination) are all acquired with the same acquisition parameters. In the basic PLI setup we typically acquire a series of images where the analyser (rotatable polariser) is at different rotation angles. We then fit a sinusoid on a pixel wise basis (wrt polarisation angle) to extract the various PLI maps. It therefore would make sense to use the aqu flag if you were to share the original images acquired with different polariser rotations, but it might not be super appropriate to use it to distinguish between the output maps?

One could of course argue that the PLI maps aren’t exactly raw data, however I think that they can be considered "minimally processed data" and are often the most useful maps to share.

The answer may be that this distinction isn’t super important when using the acq flag, but I thought it might be helpful to highlight this as potentially being a slightly odd use case.

Reply:
Thanks @amyfdh for the feedback and clarification!

I agree with you that the `acq-label` in its strict sense would better apply to the original image acquired with different polariser rotations.
In that case, I think the PLI maps would be considered "processed data" and part of a `derivatives` pipeline in BIDS.